app/assets/javascripts/submitFormAjax.js

```diff
@@ -8,10 +8,10 @@ function sendFormSubmissionAjax(dataObj, submitTo, responseEl = "", callback) {
     data: dataObj
   })
   .done((event) =>{
-    if (!!responseEl) {
+    if (responseEl) {
       $(responseEl).trigger('ajax:success', event);
     }
-    callback && callback(event);
+    if (callback) { callback(event); };
```

Comment:
```suggestion
    if (callback) { callback(event); }
    return;
```

@noi5e we can simply `return` or do we have to return output of `callback ` function?

Reply:
@Sagarpreet, no need to return the output of `callback`. This is just calling the function.

I'm pretty sure that we don't need to do an empty `return` here either&mdash;this will work without it.

I think the thing to do here is to remove the unnecessary semicolon, and just override codeclimate's other suggestion.

I know that @jywarren is able to override codeclimate and merge this!